java/api/src/main/java/org/ray/api/NodeInfo.java

```diff
@@ -1,4 +1,4 @@
-package org.ray.api.gcs;
+package org.ray.api;
```

Comment:
I think maybe we should still put this in a sub-package, but I don't have a good name at this moment.

Reply:
Considering there is only 1 `Info` class now, I think it's okay to keep it under this package. We can move it to `org.ray.api.info` package once there're some other info classes.

Reply:
I think we should avoid refactoring the api module frequently, because that will break app code. So it's worth it to come up with a good naming at the first place.
Maybe move both this class and `RuntimeContext` to `org.ray.api.runtime`?

Reply:
I don't think we should move `RuntimeConext` to `runtime` module since we should avoid exposing too many methods in `runtime` module to normal users. In other words, users should not use `(AbstructRuntime)Ray.internal()` frequently. (Correct me if I'm wrong.)

Maybe `org.ray.api.info` package is a choice.

Reply:
I meant the `org.ray.api.runtime` package (the package where `Runtime.java` resides in, still in `api` module). Not the `runtime` module.

Reply:
But that is probably not good, because normal users don't need to care about `Runtime.java`. Maybe `org.ray.api.runtime-context` is better?

Reply:
Sorry, I got it wrong. I'd like to use `org.ray.api.runtime-context`.